Conceptpower+Spring/src/test/java/edu/asu/conceptpower/app/service/impl/ConceptWrapperServiceTest.java

```diff
@@ -0,0 +1,136 @@
+package edu.asu.conceptpower.app.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.conceptpower.app.core.impl.ConceptListManager;
+import edu.asu.conceptpower.app.core.impl.ConceptTypesManager;
+import edu.asu.conceptpower.core.ConceptList;
+import edu.asu.conceptpower.core.ConceptType;
+
+public class ConceptWrapperServiceTest {
+
+    @Mock
+    private ConceptListManager conceptListManager;
+
+    @Mock
+    private ConceptTypesManager conceptTypesManager;
+
+    @InjectMocks
+    private ConceptWrapperService conceptWrapperService;
+
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void test_fetchAllConceptTypes_successForConceptLists() {
```

Comment:
why is this test called test_fetchAllConceptTypes_... if it's not testing that method?